src/thunderbots/software/util/parameter/dynamic_parameters.h

```diff
@@ -81,5 +81,26 @@ namespace Util
             extern Parameter<double> linear_sensitivity;
             extern Parameter<double> angular_sensitivity;
         }  // namespace XBoxControllerDemo
-    }      // namespace DynamicParameters
+
+        namespace NetworkInput
+        {
+            namespace Filter
+            {
+                namespace PartileFilter
+                {
+                    extern Parameter<int32_t> PARTICLE_FILTER_NUM_CONDENSATIONS;
```

Comment:
We should add a quick comment for each of these explaining what they mean. If there are any that you don't understand, Mathew would be the person to ping about this.